src/widgettoolbar.js

```diff
@@ -0,0 +1,259 @@
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import ContextualBalloon from '@ckeditor/ckeditor5-ui/src/panel/balloon/contextualballoon';
+import ToolbarView from '@ckeditor/ckeditor5-ui/src/toolbar/toolbarview';
+import BalloonPanelView from '@ckeditor/ckeditor5-ui/src/panel/balloon/balloonpanelview';
+import { isWidget } from './utils';
+
+const defaultBalloonClassName = 'ck-toolbar-container';
```

Comment:
I would not define it as here. Just use string in the method, especially since it is used once. KISS.